app/views/index.scala.html

```diff
@@ -196,46 +196,49 @@
             <div id="choropleth"></div>
         </div>
     </div>
-    <div class="container" id="difference-container">
-        <div id="dcanim-div">
-            <img src="@routes.Assets.at("assets/" + cityId + "/skyline.png")">
+    <div class="ps-skyline">
+        <div class="ps-skyline-container">
+            @if(cityName == "Washington" || cityName == "Seattle") {
+                <img src="@routes.Assets.at("assets/" + cityId + "/skyline.png")">
+            } else {
+                <img style="padding-top: 75px" src="@routes.Assets.at("assets/" + cityId + "/skyline.png")">
+            }
+            <img src="@routes.Assets.at("assets/" + "generic_cloud_overlay.png")" class="skyline-cloud-overlay">
         </div>
-        <div class="im-centered5">
-            <div class="row" style="margin: 0 auto; width: 450px;">@Messages("landing.stats.title")</div>
+        <div class="ps-skyline-stats-holder">
+            <span class="ps-skyline-stats-holder-stat-number" id="percentage">45</span>
```

Comment:
Can you actually change these default numbers to like "-1" or "NA" or "error" or something? I've never seen the numbers fail to show up, but if they did, I'd rather not have these random numbers in there. Would rather make it clear to users that there was an error.